routes/search/resolvers.js

```diff
@@ -0,0 +1,82 @@
+const docsEn = require('../../lib/i18n').docs['en-US']
+const npmPkgs = require('electron-npm-packages')
+const repos = require('repos-using-electron/lite')
+
+const searchScores = {
+  docs: [
+    { field: 'title', weight: 10 },
+    { field: 'description', weight: 3 },
+    { field: 'markdown', weight: 1 }
+  ],
+  npm: [
+    { field: 'name', weight: 10 },
+    { field: 'keywords', weight: 2 },
+    { field: 'description', weight: 5 }
+  ],
+  repos: [
+    { field: 'fullName', weight: 10 },
+    { field: 'description', weight: 5 }
+  ]
+}
+
+const trim = (arr) => {
+  // TODO: use real pagination to eliminate use of an entirely arbituary
+  // limit that's totally not the answer to life, the universe and everything
+  const limit = 42
+  if (arr.length < limit) {
+    return arr
+  }
+  return arr.splice(arr.length - limit, arr.length - limit)
```

Comment:
I think this might be equivalent and a bit easier to read: `return arr.slice(0, limit)`.. and if you use that you don't even need the length check in the lines above.